var/spack/repos/builtin/packages/hpcg/package.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+import platform
+
+
+class Hpcg(AutotoolsPackage):
+    """HPCG is a software package that performs a fixed number of multigrid
+    preconditioned (using a symmetric Gauss-Seidel smoother) conjugate gradient
+    (PCG) iterations using double precision (64 bit) floating point values."""
+
+    homepage = "https://www.hpcg-benchmark.org"
+    url = "http://www.hpcg-benchmark.org/downloads/hpcg-3.1.tar.gz"
+    git = "https://github.com/hpcg-benchmark/hpcg.git"
+
+    version('develop', branch='master')
+    version('3.1', sha256='33a434e716b79e59e745f77ff72639c32623e7f928eeb7977655ffcaade0f4a4')
+
+    variant('openmp', default=True, description='Enable OpenMP support')
+
+    patch('missing_variable.patch')
+
+    depends_on('mpi@1.1:')
+
+    arch = '{0}-{1}'.format(platform.system(), platform.processor())
+    build_targets = ['arch={0}'.format(arch)]
+
+    def configure(self, spec, prefix):
+        CXXFLAGS = '-O3 -ffast-math -ftree-vectorize -ftree-vectorizer-verbose=0 '
```

Comment:
```suggestion
        CXXFLAGS = '-O3 -ffast-math '
        CXXFLAGS += '-ftree-vectorize -ftree-vectorizer-verbose=0 '
```
This line is still too long. Maybe something like this?